.gitattributes

```diff
@@ -9,3 +9,4 @@ Documentation/helm-values.rst linguist-generated
 Documentation/codeowners.rst linguist-generated
 Documentation/_static/* -diff
 *svg -diff
+pkg/k8s/client
```

Comment:
not all the directory should be ignored IIRC only these:
```
pkg/k8s/client/clientset/
pkg/k8s/client/informers/
pkg/k8s/client/listers/
```

Reply:
youre right good catch, i'll fix that up